### DISCUSSION TITLE
Can we extract GCS score for ED patients?

### DISCUSSION THREAD

**Discussion by juhha1** [MIT-LCP/mimic-code#1227] [MIMIC-IV-ED]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * [x] Checked the online documentation: https://mimic.mit.edu/
    * [x] Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description
To extract GCS for patients who admitted to ICU, we can use mimic_icu.chartevents.
I am wondering if there is any way to extract GCS for patients who did not admit to ICU, such as patients who visited ED but not admitted to hospital?
Is this simply because recording GCS is not one of procedures done in ED?

Thanks.

**Comment by SichengH** [comment]
The informatics system is different between ED and ICU. Therefore GCS information wouldn't be extracted in the same way. I would suggest searching the ED tables if you would like to investigate them further.
